.changeset/dry-cows-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/repo": patch
+---
+
+Add iterator support to the `BlockMap` class
```

Comment:
Huh!  How did this tie in?

Reply:
Was looking for ways to optimize the operations performed during transactions.